apps/server/src/server.js

```diff
@@ -1,5 +1,4 @@
 import 'dotenv/config'
-/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
We can get rid of this `eslint-disable` if we remove the multiple `async` below, and use `void`, instead of `await`.